internal/server_control/config.go

```diff
@@ -2,21 +2,178 @@ package server_control
 
 import (
 	"context"
+	"fmt"
+	"reflect"
+	"strconv"
+	"unicode"
 
 	"github.com/Minecraft-Unified-Hub-Team/ServerControl/internal/api"
+	"github.com/fatih/structs"
 	"github.com/sirupsen/logrus"
 )
 
-func (sch *ServerControlHandler) WriteSettings(ctx context.Context, req *api.WriteSettingsRequest) (*api.WriteSettingsResponse, error) {
-	var err error = nil
+func (sch *ServerControlHandler) Update(ctx context.Context, req *api.UpdateConfigRequest) (*api.UpdateConfigResponse, error) {
```

Comment:
may be in api we should restart server after config change? but may be not... we need discuss that